File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int answer = 0;
        
        int freeDrink = returnDrink(n);
        answer = (n * 12000) + (k * 2000) - (freeDrink * 2000);
        return answer;
    }
    
    public static int returnDrink(int n) {
        double cal = 0.0;
        if( (n / 10) >= 1 ) {
            cal = (n/10);
            int freeDrink = (int) cal;
            return freeDrink;
        } else {
            return 0;
        }
    }
}

File: 프로그래머스/0/120830. 양꼬치/양꼬치.js
function solution(n, k) {
    let total = 0;
    let service = 0;
    let answer = 0;    
    let serviceCount = 0;
    
    function calService (n, k) {
        if ( n >= 10 ) {
           let i = n % 10;
            serviceCount = (n - i) / 10
        }
    }
    
    service = calService(n,k);
    total = (12000 * n) + (2000 * k) - (serviceCount * 2000)
    answer = total;
    return answer;
}